bot/exts/moderation/infraction/_utils.py

```diff
@@ -178,17 +179,18 @@ async def notify_infraction(
     embed.title = INFRACTION_TITLE
     embed.url = RULES_URL
 
-    if infr_type in APPEALABLE_INFRACTIONS:
-        embed.set_footer(text=INFRACTION_APPEAL_FOOTER)
+    embed.set_footer(
+        text=INFRACTION_APPEAL_EMAIL_FOOTER if infr_type == 'Ban' else INFRACTION_APPEAL_MODMAIL_FOOTER
```

Comment:
I'd rather a constant set of email appealable infractions, but it is fine like this

Reply:
For all infractions that remove the individual from the server, the `infr_type` argument that gets passed to this function is `'Ban'` (upper-case B), as opposed to `'ban'` or `'kick'`, which are the names for those infractions used elsewhere in the code base. I therefore decided on a string literal in the function so that we don't give the false impression that `'Ban'` is used elsewhere.

Reply:
Alright, it is fine like this then